platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/HederaLifecycles.java

```diff
@@ -14,10 +14,9 @@
  * limitations under the License.
  */
 
-package com.hedera.node.app.state.merkle;
+package com.swirlds.platform.state;
```

Comment:
seems that we should not have a class called Hedera in the platform

Reply:
Agree. It should be renamed